helpers/test-helper.js

```diff
@@ -253,5 +253,9 @@ export function bnSqrt(bn) {
       .add(a)
       .div(web3Utils.toBN(2));
   }
+  // Do we need while loop? Not sure how precise the formula above is
```

Comment:
I don't believe the loop is needed (though obviously the addition is). I believe you could also replace the addition with `b = b.addn(1)`.